FinancialTracker/Firebase.swift

```diff
@@ -161,6 +172,34 @@ class FirebaseHandler {
         self.user.balance = balance
         
         firestore.collection(usersKey).document(currentUser.uid).setData([balanceKey: balance], merge: true)
+        
+        completionHandler(nil, true)
+    }
+    
+    func addExpenseToCurrentUser(_ expenseAmount: Int, category: String, completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
+        guard let currentUser = currentUser else {
+            completionHandler(FirebaseError.access, false)
+            return
+        }
+                
+        let usersKey = DBCollectionKey.users.rawValue
+        let expensesKey = User.CodingKeys.expenses.rawValue
+        
+        let expense = Expense(amount: expenseAmount, date: String(Date.init().description.prefix(19)), category: category)
```

Comment:
what the is this `Date.init().description.prefix(19)` ??

Reply:
The idea Date.init().description returns a full date, e.g. 2021-12-31 12:31:29 +0000 with time zone, so i just take everything up to the time zone. Can be changed if needed.

Reply:
Checkout `DateFormatter` for this ..

Reply:
https://www.hackingwithswift.com/books/ios-swiftui/working-with-dates